### PR TITLE
Add new gateway - Worldpay Online Payments

### PR DESCRIPTION
**Pull request by ao**

### CODE REVIEW

File: lib/active_merchant/billing/gateways/worldpay_online_payments.rb [hunk 1]
```diff
@@ -0,0 +1,313 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class WorldpayOnlinePaymentsGateway < Gateway
+      self.live_url = self.test_url = 'https://api.worldpay.com/v1/'
+      #self.live_url = self.test_url = 'https://api.worldpay.com/v1/'
+
+      self.default_currency = 'GBP'
+      self.money_format = :cents
+
+      self.supported_countries = %w(HK US GB AU AD BE CH CY CZ DE DK ES FI FR GI GR HU IE IL IT LI LU MC MT NL NO NZ PL PT SE SG SI SM TR UM VA)
```
Comment (ao) [inline lib/active_merchant/billing/gateways/worldpay_online_payments.rb]: Yes.

**Comment by wvanbergen** [comment]
@ao Are you still working on this? Tests are still failing.

**Comment by ao** [comment]
yes

**Comment by ntalbott** [comment]
Over 2 months since any movement; going to close out. @ao please reopen for review once you've got it ready to go.

**Pull request closed without merging**